simulation/tests/test_costs.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import random
+
+import pytest
+
+from simulation.agents import Household
+from simulation.constants import (
+    BOILERS,
+    HEAT_PUMPS,
+    BuiltForm,
+    ConstructionYearBand,
+    Epc,
+    HeatingSystem,
+    OccupantType,
+    PropertyType,
+)
+from simulation.costs import (
+    get_heating_fuel_costs_net_present_value,
+    get_unit_and_install_costs,
+)
+from simulation.model import CnzAgentBasedModel
+
+
+def household_factory(**agent_attributes):
+    default_values = {
+        "location": "Test Location",
+        "property_value": 264_000,
+        "floor_area_sqm": 82,
+        "off_gas_grid": False,
+        "construction_year_band": ConstructionYearBand.BUILT_1919_1944,
+        "property_type": PropertyType.HOUSE,
+        "built_form": BuiltForm.MID_TERRACE,
+        "heating_system": HeatingSystem.BOILER_GAS,
+        "heating_system_install_date": datetime.date(2021, 1, 1),
+        "epc": Epc.D,
+        "potential_epc": Epc.C,
+        "occupant_type": OccupantType.OWNER_OCCUPIER,
+        "is_solid_wall": False,
+        "walls_energy_efficiency": 3,
+        "windows_energy_efficiency": 3,
+        "roof_energy_efficiency": 3,
+        "is_heat_pump_suitable_archetype": True,
+        "is_heat_pump_aware": True,
+    }
+    return Household(**{**default_values, **agent_attributes})
+
+
+def model_factory(**model_attributes):
```

Comment:
These can be replaced by the `from tests.common import household_factory, model_factory`